cmd/build.go

```diff
@@ -46,7 +49,10 @@ var buildCmd = &cobra.Command{
 		}
 		cmdName := "docker"
 		cmdArgs := []string{"build", "-t", buildImage, "-f", dockerfile, extractDir}
-		execAndWait(cmdName, cmdArgs, DockerLog)
+		execError := execAndWait(cmdName, cmdArgs, DockerLog)
+		if execError != nil {
+			return execError
```

Comment:
I think this should be wrapped.

Reply:
The problem with the wrapping is that it makes the stack look like it came from the caller rather than the function that has the original error.
In the future we need to discuss wrapping of original golang type errors perhaps.

Reply:
OK - let's leave this as it is right now - we'll revisit tracing in another occasion.